### DISCUSSION TITLE
Syndicate self implant machine

### DISCUSSION THREAD

**Discussion by Pullen68** [ParadiseSS13/Paradise#26977] [Documents]
So you can buy a few pretty good implants from the uplink but they centre on a very slim offering. RnD however can make a load of combat and non combat implants that can be very useful. 

The downside is you have to go to medical to get them implanted. This is fine for utility implants but a lot of questions get asked if you look at any of the combat implants. IE CNS rebooter or sec hud implant ect. These implants will get you tagged by security so fast from a doc or CMO reporting you that you will never get to use them as a tot. I feel this limity some variety for those who like implants.

My suggestion for a way to bring more spice is an uplink item that you can load implants into just like the syndi auto implant machine. This will allow you to self implant and experiment with a wider more diverse implant selection.

For balancing this item would have to have a limit to the number of implants you can use it with, maybe its a one time use maybe upto 3 different implants that it to be discussed. Same goes for the TC cost my feeling is around the 20 mark but depends on how re usable it is.

Would love to hear people's thoughts.

**Comment by PollardTheDragon** [comment]
Autosurgeons are purchasable on cybernetic revolution for a fair chunk of TC, as things like antidrop and emagged cybernetic hearts are very powerful.

Otherwise... bribe the roboticist.

**Comment by Scribble-Sheep** [comment]
Correct me if I'm wrong, but isn't it possible to just use an organ remover + a monkey to do this?

**Reply by Pullen68** [reply]
Tbh I hadn't thought of that will give it a try.
